actionview/lib/action_view/lookup_context.rb

```diff
@@ -54,24 +54,17 @@ module Accessors #:nodoc:
     register_detail(:handlers) { Template::Handlers.extensions }
 
     class DetailsKey #:nodoc:
-      alias :eql? :equal?
+      @store = Concurrent::Map.new
 
-      @details_keys = Concurrent::Map.new
+      # Make hash uniq.
+      alias_method :hash, :object_id
```

Comment:
I don't know if everything will be fine with jRuby and other rubies.
Maybe I should change this to uuid?